service/controller/resource/pvc/local_pv.go

```diff
@@ -0,0 +1,73 @@
+package pvc
+
+import (
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Note: Local volumes can only be used as a statically created PersistentVolume.
+//       Dynamic provisioning is not supported.
+func newLocalPVs(customObject v1alpha1.KVMConfig) ([]*corev1.PersistentVolume, error) {
+	var persistentVolumes []*corev1.PersistentVolume
+
+	filesystemVolumeMode := corev1.PersistentVolumeFilesystem
+
+	for _, workerNode := range customObject.Spec.KVM.Workers {
+		for _, hostVolume := range workerNode.HostVolumes {
+			var persistentVolume = &corev1.PersistentVolume{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "PersistentVolume",
+					APIVersion: "v1",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: key.LocalPVCName(key.ClusterID(customObject), hostVolume.MountTag),
+					Labels: map[string]string{
+						"app":      key.WorkerID,
+						"cluster":  key.ClusterID(customObject),
+						"customer": key.ClusterCustomer(customObject),
+						"node":     "",
```

Comment:
I would put here the MC worker node where the PV points to. Can't say how to pull the MC information in here.